client/login/magic-login/index.jsx

```diff
@@ -127,14 +127,31 @@ class MagicLogin extends Component {
 	};
 
 	componentDidMount() {
+		const { userEmail, oauth2Client, query } = this.props;
+
 		this.props.recordPageView( '/log-in/link', 'Login > Link' );
 
-		if ( isGravPoweredOAuth2Client( this.props.oauth2Client ) ) {
+		if ( isGravPoweredOAuth2Client( oauth2Client ) ) {
 			this.props.recordTracksEvent( 'calypso_gravatar_powered_magic_login_email_form', {
 				client_id: this.props.oauth2Client.id,
 				client_name: this.props.oauth2Client.title,
 			} );
 		}
+
+		// If the auto_trigger query parameter is set to true, automatically trigger the email send.
+		if ( query?.auto_trigger !== undefined ) {
```

Comment:
Minor nitpick: I believe we should only handle truthy values, e.g: `true`, `1`, etc.

Reply:
I actually wanted to support here just presence of this param, so also `?auto_trigger`, without a value. Do you think we should change it to support specific value?